akd/src/directory.rs

```diff
@@ -137,6 +144,11 @@ impl<S: Database + Sync + Send, V: VRFKeyStorage> Directory<S, V> {
             .into_iter()
             .collect::<HashMap<_, _>>();
 
+        info!(
+            "Completed VRF label computation ({} seconds elapsed)",
+            publish_start_time.elapsed().as_secs(),
```

Comment:
Does it make sense to print the elapsed time for each phase instead of from the publish start? We wouldn't need to subtract the previous phase each time then.